source/usage-examples/insertOne.txt

```diff
@@ -3,3 +3,52 @@ Insert a Document
 =================
 
 .. default-domain:: mongodb
+
+You can insert a single document in a collection by using the ``InsertOne()``
+method.
+
+The following example creates a new document containing a ``title`` and ``text``
+field nad inserts it into the ``Haikus`` collection of the ``insertDB`` database:
+
+.. note:: Non-existent Database and Collections
+
+    Databases and/or collections that don't exist will be created automatically
+    when you perform an insert operation against them.
```

Comment:
```suggestion
    The driver automatically creates databases and/or collections that don't exist
    when you perform an insert operation against them.
```
Suggestion: As is, it includes future tense and seems to be passive with "be created"